### PR TITLE
Add descirptions to these app/*/types.go files

### PR DESCRIPTION
**Pull request by tanjing2020**
#### What type of PR is this?
/kind cleanup

#### Which issue(s) this PR fixes:

Ref #99675 

#### Does this PR introduce a user-facing change?

```
NONE
```

#### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:

```docs

```

### CODE REVIEW

**Comment by nikhita** [comment]
@tanjing2020  thanks for the PR! there are a few files that need to be autogenerated when comments are updated in types.go

I have listed the scripts you need to run in the main issue body in https://github.com/kubernetes/kubernetes/issues/99675. Can you please run them?

**Comment by tanjing2020** [comment]
Sorry, I've ignored these steps and I'll do it again.

Update autogenerated files by running the following scripts:
hack/update-generated-protobuf.sh
hack/update-openapi-spec.sh
hack/update-generated-swagger-docs.sh

**Comment by nikhita** [comment]
> Sorry, I've ignored these steps and I'll do it again.

That's my bad! I forgot to add these instructions in the issue body before. :)

File: staging/src/k8s.io/api/apps/v1beta1/generated.proto [hunk 1]
```diff
@@ -325,6 +325,8 @@ message ScaleStatus {
 // The StatefulSet guarantees that a given network identity will always
 // map to the same storage identity.
 message StatefulSet {
+  // Standard list metadata.
```
Comment (CKchen0726) [inline staging/src/k8s.io/api/apps/v1beta1/generated.proto:328]: "Standard list metadata." is invalid

File: staging/src/k8s.io/api/apps/v1beta2/generated.proto [hunk 2]
```diff
@@ -613,6 +613,8 @@ message ScaleStatus {
 // The StatefulSet guarantees that a given network identity will always
 // map to the same storage identity.
 message StatefulSet {
+  // Standard list metadata.
```
Comment (CKchen0726) [inline staging/src/k8s.io/api/apps/v1beta2/generated.proto:616]: "Standard list metadata." is invalid

File: staging/src/k8s.io/api/apps/v1beta1/types_swagger_doc_generated.go [hunk 3]
```diff
@@ -197,9 +197,10 @@ func (ScaleStatus) SwaggerDoc() map[string]string {
...
-	"spec":   "Spec defines the desired identities of pods in this set.",
-	"status": "Status is the current status of Pods in this StatefulSet. This data may be out of date by some window of time.",
+	"":         "DEPRECATED - This group version of StatefulSet is deprecated by apps/v1beta2/StatefulSet. See the release notes for more information. StatefulSet represents a set of pods with consistent identities. Identities are defined as:\n - Network: A single stable DNS and hostname.\n - Storage: As many VolumeClaims as requested.\nThe StatefulSet guarantees that a given network identity will always map to the same storage identity.",
+	"metadata": "Standard list metadata. Standard object metadata; More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#metadata.",
```
Comment (CKchen0726) [inline staging/src/k8s.io/api/apps/v1beta1/types_swagger_doc_generated.go:201]: "Standard list metadata." is invalid

**Review by CKchen0726 — APPROVED** [review]

**Comment by tanjing2020** [comment]
#102034
/closed

**Pull request closed without merging**